docs/UserGuide.md

```diff
@@ -198,7 +198,7 @@ The find command has different behaviours depending on the current list:
 1. In `STUDENTS` list:
     - Finds students whose names contain any of the given keywords.
```

Comment:
Actually it finds the students whose names contains the given search string
its no longer split by multiple keywords

Reply:
Thank you, made changes in new commit.